### PR TITLE
[FR] Update `Beatmapping\Overall_difficulty`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to update the French translation of the wiki page `Beatmapping\Overall_difficulty`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
- [x] The changes have been approved on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmapping/Overall_difficulty/fr.md [hunk 1]
```diff
@@ -17,19 +16,30 @@ L'**overall difficulty** (abrégé en ***OD***) définit la difficulté d'obteni
 
 ## Timing
 
-Des valeurs d'OD plus élevées signifient une hit window plus courte pour frapper les [objets](/wiki/Hit_object), à la fois en général et pour obtenir des [scores](/wiki/Gameplay/Score) élevés. La largeur de l'intervalle pour frapper une note, centrée sur le timing correct de l'objet, est définie par le tableau suivant :
+Des valeurs d'OD plus élevées signifient une fenêtre de hit plus courte pour frapper les [objets](/wiki/Hit_object), à la fois en général et pour obtenir des [scores](/wiki/Gameplay/Score) élevés. Le hit error maximale autorisée pour chaque objet dans osu! et osu!mania, centrée sur le timing correct de l'objet, est définie par les tableaux suivants :
```
Comment (Walavouchey) [inline wiki/Beatmapping/Overall_difficulty/fr.md:19]: would "erreur de hit" also work?

Comment (DoryanR) [reply wiki/Beatmapping/Overall_difficulty/fr.md:19]: It sounds strange to me this formulation

Comment (Ganondorf-1) [inline wiki/Beatmapping/Overall_difficulty/fr.md:19]:
```suggestion
Des valeurs d'OD plus élevées signifient une fenêtre de timing plus courte pour cliquer sur les [objets](/wiki/Hit_object), à la fois de façon générale et pour obtenir des [scores](/wiki/Gameplay/Score) élevés. La fenêtre d'erreur de timing autorisée pour chaque objet dans osu! et osu!mania, centrée sur le timing correct de l'objet, est définie par les tableaux suivants :
```

File: wiki/Beatmapping/Overall_difficulty/fr.md [hunk 2]
```diff
@@ -17,19 +16,30 @@ L'**overall difficulty** (abrégé en ***OD***) définit la difficulté d'obteni
...
 
-| Score | hit window |
+### osu!
+
+| Score | Fenêtre de hit (ms) |
```
Comment (Ganondorf-1) [inline wiki/Beatmapping/Overall_difficulty/fr.md:23]:
```suggestion
| Score | Fenêtre de timing (ms) |
```

File: wiki/Beatmapping/Overall_difficulty/fr.md [hunk 3]
```diff
@@ -17,19 +16,30 @@ L'**overall difficulty** (abrégé en ***OD***) définit la difficulté d'obteni
...
+| MAX | `16` |
+| 300 | `64 - 3 * OD` |
+| 200 | `97 - 3 * OD` |
+| 100 | `127 - 3 * OD` |
+| 50 | `188 - 3 * OD` |
 
-Si le joueur frappe en dehors de la hit window du 50, cela comptera comme un miss. Si les hit windows de deux objets se chevauchent, le second objet sera inaccessible jusqu'à ce que le premier disparaisse à cause du [notelock](/wiki/Gameplay/Judgement/Notelock).
+Si le joueur frappe en dehors de la fenêtre de hit du 50, cela comptera comme un miss. Si les fenêtres de hits de deux objets se chevauchent, le second objet sera inaccessible jusqu'à ce que le premier disparaisse à cause du [notelock](/wiki/Gameplay/Judgement/Notelock).
```
Comment (Ganondorf-1) [inline wiki/Beatmapping/Overall_difficulty/fr.md:38]:
```suggestion
Si le joueur clique en dehors de la fenêtre de timing du 50, cela comptera comme un miss. Si les fenêtres de timing de deux objets se chevauchent, le second objet sera inaccessible jusqu'à ce que le premier disparaisse à cause du [notelock](/wiki/Gameplay/Judgement/Notelock).
```

File: wiki/Beatmapping/Overall_difficulty/fr.md [hunk 4]
```diff
@@ -17,19 +16,30 @@ L'**overall difficulty** (abrégé en ***OD***) définit la difficulté d'obteni
...
 
 ## Sliders et spinners
 
-Dans [osu!](/wiki/Game_mode/osu!), les sliders récompense le joueur d'un 300 tant qu'il frappe dans la hit window du 50. Cela est parfois appelé *slideracc* et est supprimé dans le [ScoreV2](/wiki/Game_modifier/ScoreV2).
+Dans [osu!](/wiki/Game_mode/osu!), les sliders récompense le joueur d'un 300 tant qu'il frappe dans la fenêtre de hit du 50. Cela est parfois appelé *slideracc* et est supprimé dans le [ScoreV2](/wiki/Game_modifier/ScoreV2).
```
Comment (Ganondorf-1) [inline wiki/Beatmapping/Overall_difficulty/fr.md:42]:
```suggestion
Dans [osu!](/wiki/Game_mode/osu!), les sliders récompensent le joueur d'un 300 tant qu'il clique dans la fenêtre de timing du 50. Cela est parfois appelé *slideracc* et est supprimé dans [ScoreV2](/wiki/Game_modifier/ScoreV2).
```

File: wiki/Beatmapping/Overall_difficulty/fr.md [hunk 5]
```diff
@@ -43,10 +53,10 @@ Il existe quatre [mods](/wiki/Game_modifier) qui modifient l'OD lorsqu'ils sont
 
 - [Easy](/wiki/Game_modifier/Easy) : Réduit de moitié la valeur de l'OD.
 - [Hard Rock](/wiki/Game_modifier/Hard_Rock) : Multiplie la valeur de l'OD par 1,4, jusqu'à un maximum de 10.
-- [Double Time](/wiki/Game_modifier/Double_Time) : La valeur de l'OD n'est pas affectée, mais en raison de l'augmentation de 50 % de la vitesse de jeu, les hit windows sont 33% plus courtes.
-- [Half Time](/wiki/Game_modifier/Half_Time) : La valeur de l'OD n'est pas affectée, mais en raison de la diminution de 25 % de la vitesse de jeu, les hit windows sont 33% plus longues.
+- [Double Time](/wiki/Game_modifier/Double_Time) : La valeur de l'OD n'est pas affectée, mais en raison de l'augmentation de 50 % de la vitesse de jeu, les fenêtres de hits sont 33% plus courtes.
```
Comment (Ganondorf-1) [inline wiki/Beatmapping/Overall_difficulty/fr.md:56]:
```suggestion
- [Double Time](/wiki/Game_modifier/Double_Time) : La valeur de l'OD n'est pas affectée, mais en raison de l'augmentation de 50 % de la vitesse de jeu, les fenêtres de timing sont 33% plus courtes.
```

File: wiki/Beatmapping/Overall_difficulty/fr.md [hunk 6]
```diff
@@ -43,10 +53,10 @@ Il existe quatre [mods](/wiki/Game_modifier) qui modifient l'OD lorsqu'ils sont
...
+- [Half Time](/wiki/Game_modifier/Half_Time) : La valeur de l'OD n'est pas affectée, mais en raison de la diminution de 25 % de la vitesse de jeu, les fenêtres de hits sont 33% plus longues.
```
Comment (Ganondorf-1) [inline wiki/Beatmapping/Overall_difficulty/fr.md:57]:
```suggestion
- [Half Time](/wiki/Game_modifier/Half_Time) : La valeur de l'OD n'est pas affectée, mais en raison de la diminution de 25% de la vitesse de jeu, les fenêtres de timing sont 33% plus longues.
```

File: wiki/Beatmapping/Overall_difficulty/fr.md [hunk 7]
```diff
@@ -43,10 +53,10 @@ Il existe quatre [mods](/wiki/Game_modifier) qui modifient l'OD lorsqu'ils sont
...
 
-Alors que les mods Half Time *(HT)* et Double Time *(DT)* ne modifient pas la valeur de l'OD, la différence de vitesse y entraîne un changement dans les hit windows. Comme la mise à l'échelle est différente pour chaque valeur de score, le DT fait que les intervalles pour 100 et 50 sont plus resserrés que d'habitude par rapport au 300, et le HT les fait devenir plus indulgents. 
+Alors que les mods Half Time *(HT)* et Double Time *(DT)* ne modifient pas la valeur de l'OD, la différence de vitesse y entraîne un changement dans les fenêtres de hits. Comme la mise à l'échelle est différente pour chaque valeur de score, le DT fait que les intervalles pour 100 et 50 sont plus resserrés que d'habitude par rapport au 300, et le HT les fait devenir plus indulgents.
```
Comment (Ganondorf-1) [inline wiki/Beatmapping/Overall_difficulty/fr.md:59]:
```suggestion
Alors que les mods Half Time *(HT)* et Double Time *(DT)* ne modifient pas la valeur de l'OD, la différence de vitesse y entraîne un changement dans les fenêtres de timing. Comme la mise à l'échelle est différente pour chaque valeur de score, le DT fait que les intervalles pour 100 et 50 sont plus resserrés que d'habitude par rapport au 300, et le HT les fait devenir plus indulgents. 
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Biokax — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**